app/build.gradle

```diff
@@ -254,8 +261,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
Was this change intended ?

Reply:
Actually, i was facing some issue with project build earlier hence I modified it but now it seems to work fine. I'll revert the changes